intel/base/gpu/ubi9-python-3.9/Dockerfile

```diff
@@ -15,6 +15,8 @@ USER 0
 WORKDIR /opt/app-root/src
 
 RUN . /etc/os-release && \
+    #TODO: Remove explicit declaration of VERSION_ID once available on version 9.4
+    VERSION_ID=9.3 && \
```

Comment:
Wouldn't be a more clear solution to actually keep using 9.3 image in our base image - https://github.com/opendatahub-io/notebooks/blob/main/base/ubi9-python-3.9/Dockerfile#L1.

I know that it affects all other our images - but this is the example why we shouldn't use the `latest` tag in our images - this is update from 9.3 to 9.4 automatically for us :thinking:

Reply:
This is a great idea, we can create an automation to do that on the base. 
I will close this PR since the owner of the image incorporate this change